src/modules/fundingManager/bondingCurveFundingManager/BondingCurveFundingManagerBase.sol

```diff
@@ -253,6 +280,98 @@ abstract contract BondingCurveFundingManagerBase is
         emit TokenDecimalsUpdated(oldDecimals, tokenDecimals);
     }
 
+    /// @dev Returns the collateral and issuance fee percentage retrieved from the tax manager for
+    ///     buy operations
+    /// @return collateralTreasury The address the protocol fee in collateral should be sent to
+    /// @return issuanceTreasury The address the protocol fee in issuance should be sent to
+    /// @return collateralBuyFeePercentage The percentage fee to be collected from the collateral
+    ///     token being deposited for minting issuance, expressed in BPS
+    /// @return issuanceBuyFeePercentage The percentage fee to be collected from the issuance token
+    ///     being minted, expressed in BPS
+    function _getBuyFeesAndTreasuryAddresses()
```

Comment:
This may be nitpicking, but can't this function be combined with `_getSellFeesAndTreasuryAddresses()` into one that receives the function selector (f.ex. `_getFunctionFeesAndTreasuryAddresses()`)? That way, we avoid duplicated code and open the door for inheriting contracts to query fees of additional functions

Reply:
Maybe it could even be moved to the base Module contract (although I'm not sure if I'm getting ahead of myself there)

Reply:
You're right, that would be more clean